### ISSUE TITLE
Stricter Ruleset for Security vs Commodity Cryptos

### ISSUE THREAD

**Issue by M1100010** [responsible-financial-innovation-act22/RFIA-bill#10]
I don't have the time or legal understand to read the bill, but from what I understand, it makes most cryptos commodities.  
I believe most cryptos should be considered securities, because they are more centeralized than a commodity should be. A commodity crypto should be:  
1. Fair launched, meaning no coins or tokens were generated as part of the supply for the creators or foundations before the public was able to earn them through some form of Proof of Work mining
2. Proof of work, meaning coins are tokens are earned through an activity that is open to all to participate in, competitive in seeking rewards, and cryptographically verifies their effort
3. Decentralized, meaning no one entity is able to unilaterally affect change to the underlying distributed ledger technology, which includes changes to its code or day to day operations.  

For example, BTC, LTC, DOGE, and XMR fit this criteria. ETH, ADA, BCH, and most other cryptos would not meet these criteria.  
My reasoning for this is that:  
1. Any premined token will have the expectation of profit from the efforts of a 3rd party because holders rely on the preminer to use the premined tokens to promote, fund development, and bring in new investors.
2. Proof of stake or other similar consensus and distribution mechanisms give holders the expectation of profit from the actions of the largest stakers, who can easily influence the operations of the distributed ledger, and must be trusted to protect the value of the crypto.
3. Centeralized coins lead by some founder or organization are most directly giving holders the expectation of profit from the actions of a 3rd party. Even though they may have a few other parties operating nodes, if they are able to convince all of them to change their node code uniformly and consistently, they are essentially acting as a single entity.  

The cryptos that don't fit these criteria for a commodity are the ones most often running off with investor funds after an uptick in price action, or being manipulated to transfer value from retail investors to the entities that control them. The public should not be led to believe that these are commodities.

**Comment by jonmarrs** [comment]
I agree. I think there should be a Decentralization Test for cryptocurrencies, similar to how there is a Howey Test for securities. Decentralized cryptocurrencies such as Bitcoin should be treated as money and should be tax-free. Centralized cryptocurrencies (the vast majority of altcoins) should be treated as property or securities and should be taxed and subject to reporting/disclosure requirements.

**Comment by jonmarrs** [comment]
Some criteria for determining whether a cryptocurrency is centralized vs. decentralized:

Decentralized:
- Proof-of-Work
- No pre-mine
- Nodes are cheap (<$1,000) and can run in homes with readily available bandwidth (<1 Gigabit) and power (<1 kW) and disk space (<2 TB). For example, you can run a node on a cheap Raspberry Pi computer at home.
- High node count (>1,000)

Centralized:
- Proof-of-Stake
- Pre-mined / founder award
- Nodes are very expensive (>$1,000) and can basically only run in centralized data centers with high bandwidth (>1 Gigabit) and power (>1 kW) and disk space (>>2 TB).
- Low node count (<1,000)

**Comment by M1100010** [comment]
>  Decentralized cryptocurrencies such as Bitcoin should be treated as money and should be tax-free.  

As much as I would like this it will probably take some time and price stabilization for this to be introduced into law. Currently the US taxes exchanges of commodities like gold, which would ideally be treated as money. It even taxes direct exchanges between gold and silver because these aren't "like for like", because gold is a "money metal" and silver is an "industrial material".  

> Some criteria for determining whether a cryptocurrency is centralized vs. decentralized:
> 
> Decentralized:
> 
>     * Proof-of-Work
> 
>     * No pre-mine
> 
>     * Nodes are cheap (<$1,000) and can run in homes with readily available bandwidth (<1 Gigabit) and power (<1 kW) and disk space (<2 TB). For example, you can run a node on a cheap Raspberry Pi computer at home.
> 
>     * High node count (>1,000)  

Adding to this list, decentralized nodes should not have automatic software updates.

**Comment by WCWedin** [comment]
The distinction here between proof of work and proof of stake seems misplaced to me, given that at scale, all proof of work translates to proof of stake in physical space in the form of computing hardware, which is ultimately purchased with fiat currency. Proof of stake simply cuts out the middleman, allowing miners to stake their coins directly, without having to exchange them for cash and then physical goods. In either case, as has been evidenced by prior Bitcoin mining pool crises, it's the holder of the most currency who holds the most control.

Further, the case often made that the labor of establishing and maintaining a mining facility is what grants cryptocurrency real value completely disregards the elementary fact of labor economics that labor that doesn't produce use-value is not really labor at all, but merely wasted effort and resources. (It also ignores the fact that much of the labor that goes into manufacturing an ASIC is slave labor in mineral mines, which, apart from being a significant moral concern for the cryptocurrency space, is also an unpriced externality that risks exposure to volatility in the cost of the labor used.)

In other words, forming a legal distinction between categories of "centralized" and "decentralized" based partially upon a technical detail that constructs a distinction without much of a real-world difference is probably ill-advised.

**Comment by frabaghe** [comment]
> all proof of work translates to proof of stake in physical space in the form of computing hardware, which is ultimately purchased with fiat currency

This is a mischaracterization. When a project controls a majority of Proof of Stake tokens, they have _unimpeachable_ control over that network. In Proof of Work, anyone who musters enough hardware can overthrow the network. The ability to overthrow control authority makes Proof of Stake wildly different from Proof of Stake. Proof of Stake is essentially equity.

**Comment by Bhaney44** [comment]
According to the United States Supreme Court, “The test of an investment contract within Securities Act is whether [the] scheme involves an investment of money in a common enterprise with profits to come solely from efforts of others, and, if test is satisfied, it is immaterial whether enterprise is speculative or nonspeculative or whether there is a sale of property with or without intrinsic value.” In fact, [Securities and Exchange Commission v. W.J. Howey Co et al.](https://supreme.justia.com/cases/federal/us/328/293/), establishes more than seven decades of precedent that to be a security pursuant to the Securities Act, the asset must be designed to produce `profits solely from the efforts of others` as part of a `common enterprise`. Thus, most digital assets are not investment contracts under the Securities Act because most digital assets do not produce any `profits solely from the efforts of others` and often lack a `common enterprise`. 

Probably the biggest problem with this is Bill is the lack of updated definitions for digital assets with respect to securities. In many cases, digital assets may be neither securities nor commodities. It's important to respect that decentralized ledger technologies are relatively new and should not be forced within existing syntactic legal structures. As is the case now, regulators and judges are making rules up - and the law - on the fly, which creates great uncertainty for compliance, while disproportionately and negatively impacting small businesses. The law needs to be set from the start, not made up _ad hoc_ according to the desire of unelected regulators. More simple definitions should be adopted to clearly define the difference between security tokens and non-security tokens. 

> Digital asset securities: only those assets which represent an equity interest in a company or common enterprise and are sold as investments.
> 
> Non-security digital assets: any asset which is intentionally decentralized, intended for use within a product or service, or operates as a cryptographic key.

**Comment by WCWedin** [comment]
The mere existence of 51% attacks put to lie the entire premise of what you're saying. The fact is that these assets are centralized not by code but instead by first-mover and capital advantage regardless. Proof of work is, in practice, nothing but an expensive way to kick the can down the road, pretending there is no king because "the market" plays the kingmaker. Bitcoin may be "too big to fail" for now, but the future is a big place, and privileging the most destructive form of a public ledger as a tax-free "currency" when its promise of decentralization is teetering on the edge of reality would be, frankly, insane.

**Comment by M1100010** [comment]
> The distinction here between proof of work and proof of stake seems misplaced to me, given that at scale, all proof of work translates to proof of stake in physical space in the form of computing hardware, which is ultimately purchased with fiat currency. Proof of stake simply cuts out the middleman, allowing miners to stake their coins directly, without having to exchange them for cash and then physical goods. In either case, as has been evidenced by prior Bitcoin mining pool crises, it's the holder of the most currency who holds the most control.

Proof of stake is a plutocratic system, which requires that the founders premine coins, and have to be trusted not to give themselves the lion's share.  
Mining is a competitive and innovative business. It drives miners to seek clean and efficient energy sources, like renewables and fugitive energy like flared methane.
Miners don't control the blockchain as stakers do because they are separate entities from node operators in a PoW system. Miners have the ability to gain and lose portion of the hashrate based on their efficiency. Mining isn't wasted work either, as the amount of energy used to mine is equal to the amount of energy it would take to attack the network. With PoS, the network could be attacked with a large loan. PoS is the equivalent to a power strip plugged into itself, because its value and security depend on it having value, while PoW is able to monetize energy that would otherwise be wasted, and use existing hardware at low cost until its value is established.

**Comment by Bhaney44** [comment]
> The mere existence of 51% attacks put to lie the entire premise of what you're saying. The fact is that these assets are centralized not by code but instead by first-mover and capital advantage regardless. Proof of work is, in practice, nothing but an expensive way to kick the can down the road, pretending there is no king because "the market" plays the kingmaker. Bitcoin may be "too big to fail" for now, but the future is a big place, and privileging the most destructive form of a public ledger as a tax-free "currency" when its promise of decentralization is teetering on the edge of reality would be, frankly, insane.

No. There is no tax-free currency. See [26 U.S.C. 1 - 1564](https://www.law.cornell.edu/uscode/text/26/subtitle-A).

**Comment by namcios** [comment]
> Thus, most digital assets are not investment contracts under the Securities Act because most digital assets do not produce any `profits solely from the efforts of others` and often lack a `common enterprise`.

This premise is flawed because all digital assets with a pre-mine or initial token sale inherently include expectation of profits. "Let's all buy this token because its price appreciation will benefit founders, developers, and 'the community.'"

Moreover, an ICO (an acronym which most projects have refrained from using but are just doing the same thing with different names) is not that different from an IPO. The tokens do provide expectation for profits as cited above as well as allow _stake_ in the project – equity. Holding coins in PoS elevate users to a different level wrt those who don't, and the amount further skews that.

The "common enterprise" in the space refers to the founding team and/or foundation that exert direct influence over the protocol indefinitely, being able to stir and control the future of the project. Large coin holders and founders in PoS are therefore analogous to a board in a common enterprise.
